roles/kubernetes-apps/csi_driver/vsphere/templates/vsphere-csi-controller-deployment.yml.j2

```
@@ -62,6 +62,8 @@ spec:
             - "--leader-election"
             - "--kube-api-qps=100"
             - "--kube-api-burst=100"
+          resources:
+            {{ vsphere_csi_attacher_resources | default({}) | to_nice_yaml | trim | indent(width=12) }}
```

Don't we need to put a condition here like
```suggestion
{% if vsphere_csi_attacher_resources | length > 0 %}
          resources:
            {{ vsphere_csi_attacher_resources | default({}) | to_nice_yaml | trim | indent(width=12) }}
{% endif %}
```
to avoid putting `resources:` when the vsphere_csi_attacher_resources is empty?

---

Is it a problem if fields are empty? I tested it works well in both cases, empty resources and not.
I prefer using it without if statements, but if you think this is better, I will rework it